src/main/software/amazon/event/ruler/Range.java

```diff
@@ -108,11 +109,11 @@ static byte[] digitSequence(byte first, byte last, boolean includeFirst, boolean
 
     private static int getHexByteIndex(byte value) {
```

Comment:
@jonessha what do you think? I feel like using the chars helps with the clarity , and then only one obscure constant is needed

Reply:
This looks good! Thank you!